ccdproc/combiner.py

```diff
@@ -150,15 +149,12 @@ def scaling(self):
 
         Parameters
         ----------
-
-        scale : function or array-like or None, optional
+        scale : function or `numpy.ndarray`-like or None, optional
```

Comment:
I'm OK with numpy.ndarray everywhere else (though would never associate it with array.array in a package that does numerics), but personally prefer the short names only for unquestionable classes like MaskedArray, I can always click on the link if in doubt.

Either way it's good to see that the docs becomes more self-consistent.